index.html

```diff
@@ -12,6 +12,14 @@
       <header>
         <a href="#" class="logo">karan</a>
         <img src="SVG/Icon - Menu.svg" alt="" class="menuIcon" />
```

Comment:
- [ ] Kindly reduce the `height` of the header to make it match Figma

## Your design:
![image](https://user-images.githubusercontent.com/50721479/194117767-ffa35790-5631-4423-b87a-84bb4380d83d.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/194117805-29addb3a-ef76-4a00-a711-781f8e5c42fd.png)